mars/services/storage/api/oscar.py

```diff
@@ -145,9 +146,11 @@ async def delete(self, data_key: str, error: str = 'raise'):
             self._session_id, data_key, error=error)
 
     @extensible
-    async def prefetch(self,
-                       data_key: str,
-                       level: StorageLevel = StorageLevel.MEMORY):
+    async def fetch(self,
+                    data_key: str,
+                    level: StorageLevel = StorageLevel.MEMORY,
+                    band: str = None,
```

Comment:
Use `band_name` instead of `band` as `band` usually refer to address-name tuples?